manifests/t/TroyeGuo/KoodoReader/1.4.0.811/TroyeGuo.KoodoReader.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TroyeGuo.KoodoReader
+PackageVersion: 1.4.0.811

```

```suggestion
PackageVersion: 1.4.0
```
This must match the version shown in control panel. Don't forget to update the version folder name when this changes